package.json

```diff
@@ -11,9 +11,9 @@
   },
   "dependencies": {
     "grunt": "^0.4.5",
-    "grunt-sass": "^1.0.0",
-    "grunt-contrib-less": "1.3.0",
+    "grunt-contrib-less": "^1.3.0",
```

Comment:
Is this just an alphabetical order thing for readability? If so, might as well commit it to master to reduce complexity of this PR.